lib/http-bridge/test/unit/Cardano/Wallet/HttpBridge/BinarySpec.hs

```diff
@@ -58,6 +62,9 @@ import qualified Data.ByteString.Lazy as BL
 import qualified Data.ByteString.Lazy.Char8 as L8
 
 {-# ANN spec ("HLint: ignore Use head" :: String) #-}
+
+type N = 'Testnet
```

Comment:
@KtorZ I added it here when I was experimenting, but I take it that it should be here anymore.

Reply:
Indeed. Type-aliases (especially this one) usually provides very little value. They make deciphering of code harder and have a tendency to spread.